src/components/CaptureContainer/index.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import * as a1lib from "@alt1/base";
+import { useState } from "react";
+import { CapturedImage } from "../CapturedImage";
+import "./CaptureContainer.scss";
+import { findHomeport } from "../../utils/homeport";
+
+export const CaptureContainer = () => {
+  const [errorText, setErrorText] = useState<string>("");
```

Comment:
You can omit the <string> here and in the next `useState` calls, it will be inferred automatically